sml_small/utils/pandas_wrapper.py

```diff
@@ -0,0 +1,233 @@
+"""
+Defines a wrapper to allow the Totals and Components method to be run with Pandas.
```

Comment:
Probably best to tweak this comment to show it is more than just totals and components

Reply:
Amended